cmd/di.go

```diff
@@ -416,3 +416,8 @@ func (di *Dependencies) bootstrapLocationComponents(options node.OptionsLocation
 	di.LocationDetector = location.NewDetector(di.IPResolver, di.LocationResolver)
 	di.LocationOriginal = location.NewLocationCache(di.LocationDetector)
 }
+
+var corsConfig = tequilapi.CorsConfig{
+	DefaultTrustedOrigin:  "https://mysterium.network",
+	AllowedOriginSuffixes: []string{"mysterium.network", "localhost"},
```

Comment:
Too many domains, lets trust just what we need

Reply:
Another thing - hardcoding trusted values without possibility to override them on startup - is a breaking change for existing api clients.

Reply:
The idea behind having to trust the whole "mysterium.network" is that we don't need to update it every time we want to add or change a new sub-domain, i.e. adding another wallet environment - but anyways, changed it to specific sub-domains because I don't see big difference :D

As for posibility to override those values - created a separate task for that: https://github.com/mysteriumnetwork/node/issues/758
It is a breaking change, but I believe that it's worth breaking supporting unknown websites :D

Reply:
It's a security issue, we need to estimate risk of ANY `localhost` talking to API

Reply:
Some site can decide to register your identity for you

Reply:
"localhost" is needed just for `--localnet` case, lets add new CLI argument `--wallet.address`

Reply:
"localhost" is needed always - Electron makes requests using browser, so it respects CORS.

Reply:
PR seems good as CORS engine feature, needs final dots on i to finalise final security